packages/theme-samples/src/DarkCustomizations.ts

```diff
@@ -0,0 +1,52 @@
+import { createTheme, ICustomizations, IPalette, ITheme } from 'office-ui-fabric-react';
+import { addVariants } from '@uifabric/variants';
+
+const DarkDefaultPalette: Partial<IPalette> = {
```

Comment:
These are the colors specified by the fluent toolkit?

Reply:
Yep! The current Dark theme toolkit